src/ape_ethereum/ecosystem.py

```diff
@@ -167,6 +168,12 @@ class Block(BlockAPI):
     difficulty: int = 0
     total_difficulty: int = Field(0, alias="totalDifficulty")
 
+    # Type re-declares.
```

Comment:
Is this just to re-arrange the order it serializes as?

Reply:
no, the `BlockAPI` uses `Any` as the type, but in `ape-ethereum`, we can safely assume it is always going to be HexBytes`.

Reply:
tbh, i dont remember why they are Any in `BlockAPI`; maybe we can be rid of that

Reply:
I think bytes/HexBytes is probably an appropiate assumption for those fields abstractly

Reply:
im good with making the change now, shouldnt really break anything except maybe the already broken starknet